javatools_launcher/build.gradle.kts

```diff
@@ -19,6 +19,8 @@ val launcherExecutableDir = layout.projectDirectory.dir("src/main/resources/exe"
 
 val processResources by tasks.registering(Copy::class) {
     from(files(launcherExecutableDir))
+    // TODO: This may cause a warning in IDEA with non-delegated compilation, but it is safe.
```

Comment:
I don't understand this "TODO". If that is safe, what is there to do? What is the action that is necessary?

Reply:
It is there to make users aware of that this is the reason behind the yellow scary looking alert thst can't be disabled, but is tracked as to be fixed by JetBrains and that it is not cause for concern. I hope that if someone notices this warning in IntelliJ and actually just clicks on the warning, so that intellij jumps to this build script, then just maybe I won't get the question "Have you seen this warning in IntelliJ? Why does it do that? Should it do that? Our intellij build is broken?" 5-10 times. This has already been the case for the "borrow the resource path from lib_ecstasy in javatools to make javatools is runnable as a Java app config in IntelliJ" warning and I keep explaining, but the question keeps being asked.

Reply:
I see. Maybe simply change "TODO" to "NOTE:"